compute/admin_guide/configure/agentless-permission-templates.adoc

```diff
@@ -0,0 +1,68 @@
+== Agentless Permission Templates
+
+Agentless scanning requires specific permissions to work on your self-hosted deployments.
+Different platforms have different permission requirements. 
+To make applying the right permissions easier, you can download permission templates for your specific platforms.
```

Comment:
Can we add the below please here in intro -- 
"There are two ways you can set up agentless scanning with Prisma Cloud.
Scan all hosts of a cloud account within the same cloud account - Independent Scanning. In this mode, the scanners are spin up in every account that is scanned. 
or
Scan all hosts of a cloud account, called target account(s), in a dedicated cloud account, called hub account. - Scan hub setup. In this mode, the scanners are spin up only in the hub account and not the target accounts being scanned by hub.